cdap-docs/admin-manual/source/installation/security.rst

```diff
@@ -270,43 +286,58 @@ LDAP Authentication
 You can configure CDAP to authenticate against an LDAP instance by adding these
 properties to ``cdap-site.xml``:
 
-========================================================== ==================================================================
-Property                                                   Value
-========================================================== ==================================================================
-``security.authentication.handlerClassName``               ``co.cask.cdap.security.server.LDAPAuthenticationHandler``
-``security.authentication.loginmodule.className``          ``co.cask.cdap.security.server.LDAPLoginModule``
-``security.authentication.handler.debug``                  ``true/false``
-``security.authentication.handler.hostname``               ``<hostname>``
-``security.authentication.handler.port``                   ``<port>``
-``security.authentication.handler.userBaseDn``             ``<userBaseDn>``
-``security.authentication.handler.userRdnAttribute``       ``<userRdnAttribute>``
+========================================================== =========================== ======================================
+Property                                                   Default Value               Description
```

Comment:
Same here, "Default Values" does not really work -- the first two are the specific entries that are needed, the remaining ones (except for the debug setting) do not have default values.